quantum/process_keycode/process_combo.c

```diff
@@ -129,9 +127,11 @@ static inline void release_combo(uint16_t combo_index, combo_t *combo) {
         keyrecord_t record = {
             .event =
                 {
-                    .key     = COMBO_KEY_POS,
-                    .time    = timer_read() | 1,
+                    .key     = {.col = 0, .row = 0},
```

Comment:
Wouldn't that somehow conflict with the left topmost key (or whatever key is wired to be row=0, col=0)? There aren't keyboards with 255 rows and columns but there most certainly are keyboards with at least one row and one column.

Reply:
My hope would be that the core code respects the newish `.event` flag and doesn't try to evaluate the keyevent. If not this would also lead to an out of bounds access on the current keymap if it tried because I doubt that there is anything mapped.